templates/generic.php

```diff
@@ -29,7 +31,8 @@
 			and put it in templates and have your BaseView automatically load it from <code>templates/views/</code>.
 		</p>
 		<p>
-			But that's just me... do what you want.
+			These are the only clues I will give you. Good luck.
```

Comment:
Extremely helpful, this should get added immediately. 💯

Reply:
Approved - this is certainly a more accurate reflection of our current best practices.